spark-indexer/src/main/java/org/uniprot/store/spark/indexer/taxonomy/TaxonomyLineageReader.java

```diff
@@ -61,7 +66,7 @@
                             .replace("{start}", "" + range[0])
                             .replace("{end}", "" + range[1]);
 
-            System.out.println("SQL: " + sql);
+            log.info("SQL: " + sql);
```

Comment:
How do we know that range has value at at index 0 and 1?

Reply:
Because we do the count first.